packages/consumption/test/modules/requests/RequestsIntegrationTest.ts

```diff
@@ -630,6 +649,88 @@ export class RequestsWhen {
         );
     }
 
+    public async iSentAnOutgoingRequestWithDeleteAttributeRequestItems(): Promise<void> {
+        const sOwnSharedIdentityAttribute1 = await this.context.consumptionController.attributes.createAttributeUnsafe({
+            content: IdentityAttribute.from({
+                value: {
+                    "@type": "BirthName",
+                    value: "A first name"
+                },
+                owner: this.context.currentIdentity
+            }),
+            shareInfo: {
+                peer: CoreAddress.from("peer"),
+                requestReference: CoreId.from("shareRequestReference1")
+            }
+        });
+
+        const predecessorId = await ConsumptionIds.attribute.generate();
+        const sOwnSharedIdentityAttribute2 = await this.context.consumptionController.attributes.createAttributeUnsafe({
+            content: IdentityAttribute.from({
+                value: {
+                    "@type": "BirthName",
+                    value: "A second name"
+                },
+                owner: this.context.currentIdentity
+            }),
+            shareInfo: {
+                peer: CoreAddress.from("peer"),
+                requestReference: CoreId.from("shareRequestReference2a")
+            },
+            succeeds: predecessorId
+        });
+
+        await this.context.consumptionController.attributes.createAttributeUnsafe({
+            id: predecessorId,
+            content: IdentityAttribute.from({
+                value: {
+                    "@type": "BirthName",
+                    value: "A second-name"
```

Comment:
More descriptive please. Make it clear that the `value` is an outdated version of the `"A second name"`.